tests/0081-admin.c

```diff
@@ -2817,12 +3025,77 @@ static void do_test_ListConsumerGroups(const char *what,
                         }
                 }
         }
+
         TEST_ASSERT(found == TEST_LIST_CONSUMER_GROUPS_CNT,
                     "expected to find %d"
                     " started groups,"
                     " got %" PRIusz,
                     TEST_LIST_CONSUMER_GROUPS_CNT, found);
 
+        if (has_match_types) {
+                /* With the options of the consumer protocol we should get 0
+                 * valid cnt */
+                rd_kafka_AdminOptions_t *optionwithconsumer =
+                    rd_kafka_AdminOptions_new(
+                        rk, RD_KAFKA_ADMIN_OP_LISTCONSUMERGROUPS);
+                rd_kafka_consumer_group_type_t consumer_type =
+                    RD_KAFKA_CONSUMER_GROUP_TYPE_CONSUMER;
```

Comment:
Yes, these are true but since the test is similar for other clients which do not have has_match_types functionality like in librdkafka, so only these were agreed upon. Otherwise, we would diverge a lot !

Reply:
I agree with @emasab. This test will fail when the consumer is created with CLASSIC protocol. Please fix this.